launch/interact.go

```diff
@@ -66,13 +70,30 @@ func (d *Interact) Run(c *exec.Cmd, commands [][]string) error {
 	// Copy stdin to the pty and the pty to stdout.
 	go func() {
 		for _, v := range commands {
-			v = append(v, "\n")
-			_, _ = io.Copy(ptmx, strings.NewReader(strings.Join(v, " ")))
-		}
 
+			v := append(v, "\n")
+			command := strings.Join(v, " ")
+			if len(command) > maxByte {
+				for {
+					if len(command) <= maxByte {
+						io.Copy(ptmx, strings.NewReader(command[:]))
+						// Wait send the command.
+						time.Sleep(time.Second * 1)
+						break
+					} else {
+						io.Copy(ptmx, strings.NewReader(command[:maxByte]))
+						// Wait send the command.
+						time.Sleep(time.Second * 1)
```

Comment:
Does it need to sleep?
I think it doesn't need to sleep for 1 second. You can use `time.Millisecond`.

Reply:
Strangely enough, it is necessary.
If you write too quickly, the string sent to shell will be broken off.
I tried the following case.
```
1ms
5ms
30ms
80ms
100ms
300ms
500ms 
```
Apparently, it's stable at over 300ms

Use `time.Millisecond` 👍